index.html

```diff
@@ -1,16 +1,101 @@
-<!DOCTYPE html>
-<html lang="en">
 
-<head>
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
-
+    <title>Cakes Project</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
 <body>
-    <!-- Add your markup here -->
+  <div class="wrapper">
+  <!--header-->
+    <header class="header">
+      <img class="logo" src="/img/cake-logo-design-template-download_529200-598.webp">
+      <p>The best cakes in town delivered to your door.</p>
+    </header>
+    <nav class="nav-section">
+      <ul>
+        <li> <a class="nav-link active" href="#">HOME</a></li>
+        <li> <a class="nav-link active" href="#">CAKES</a></li>
+        <li> <a class="nav-link active" href="#">ORDERIN</a></li>
+        <li> <a class="nav-link active" href="#">LESSONS</a></li>
+        <li> <a class="nav-link active" href="#">ABOUT</a></li>
+      </ul>
+    </nav>
+  <!--end header-->
+ <!-- top container-->
+  <section class="top-container"> 
+    <div class="main-img">
+    <img class="main-cake" src="/img/pexels-vidal.jpg">
+  </div>
+  <div class="main-paragrafo">
+    <h1>Welcome</h1>
+    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Tempora perferendis delectus soluta dolorem magnam rerum itaque blanditiis quia, nemo expedita quaerat!</p>
+  </div>
+  </section>
+  <!--portfolio-->
+  <section class="portfolio">
+         
+      <img class="images1" src="/img/pexels-daria-shevtsova.jpg" alt="#"> 
```

Comment:
It would be best to add alt text explaining what the the image is if possible :)